torBot.py

```diff
@@ -155,9 +164,11 @@ def main(conn=False):
             links = getweblinks.get_links(soup=html_content, ext=args.extension, live=args.live)
             if args.save:
                 savefile.saveJson("Links", links)
+            if args.database:
```

Comment:
Could you add a print statement either here or inside of `saveToDatabase` to notify the user that we're beginning the insertion of the links into the DB?